charts/aws-ebs-csi-driver/templates/csidriver.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1
+apiVersion: {{ ternary "storage.k8s.io/v1" "storage.k8s.io/v1beta1" (semverCompare ">=1.19.0-0" .Capabilities.KubeVersion.Version) }}
```

Comment:
I thought this became [GA in 1.18](https://kubernetes-csi.github.io/docs/csi-driver-object.html)

Reply:
Yeah, v1beta1 is deprecated in 1.19 so I used 1.19 here.. 
I can change it to 1.18 to sync with your PR, or do you want to cherry-pick the helm chart update to the release branch after merge? Then I will close this on as well

Reply:
Go ahead and change it to 1.18 just to get it fixed in the release branch faster.  Thanks

Reply:
Updated, can I get a lgtm?